experimenter/experimenter/experiments/admin.py

```diff
@@ -280,6 +280,9 @@ class NimbusExperimentAdminForm(forms.ModelForm):
     conclusion_recommendation = forms.ChoiceField(
         choices=NimbusExperiment.ConclusionRecommendation.choices, required=False
     )
+    qa_status = forms.ChoiceField(
+        choices=NimbusExperiment.QAStatus.choices, required=False
```

Comment:
I think we shoud set a default here as `Not Set`

Reply:
because your migration only sets this value for the existing experiments in the DB, for the future experiments, if they don't select the value, we can set `Not set` as a default value

Reply:
I think it also might have to be requried=True

Reply:
It should be `required=False` for the admin, I'm pretty sure. All of the fields are not required here, and I'm pretty sure you don't want to have to set `qa_status` every time you make a change. I also don't think the admin has to set a default, since that is set in the [model](https://github.com/mozilla/experimenter/pull/10055/files#diff-181caffbf9f9e1237da4cac08b35076e64e1321a12ee1935c61c7af2c155bf25) 🤔  I'm also not super familiar with the admin, so pls correct me if I'm wrong

Reply:
I am looking at the django docs if set a default on model, will be used in admin as well or not

Reply:
yup just checked the docs, if we have the value default defined on the model, then admin will recreate and use those models default and pre-populate the default value, @eliserichards also have verified this locally, so all set 💯